src/components/availability-panel/drag-drop-context/DraggableComponent.tsx

```diff
@@ -33,6 +32,13 @@ const DraggableComponent: FC<draggableProps> = ({
   title = '',
 }) => {
   const draggableIds = useContext(disableDrag);
+  const [draggableImage, setDraggableImage] = useState<string>('');
+
+  useEffect(() => {
+      getAssigneeProfilePic(draggableId).then((res) => {
+        setDraggableImage(res);
+      });
+  }, [])
```

Comment:
This API will also be called when the component unmounts, this will hit the performance

Reply:
@bhtibrewal are you sure API will be called again when component `umounts`, not on mounting-again?

Reply:
i am not able to see @bhtibrewal  in network section can you please send me the videos/images ?